desktop_version/src/Script.cpp

```diff
@@ -3514,11 +3523,13 @@ bool scriptclass::loadcustom(const std::string& t)
                     add("text(blue,0,0,"+words[1]+")");
                 break;
             }
-            int ti=help.Int(words[1].c_str());
-            int nti = ti>=0 ? ti : 1;
-            for(int ti2=0; ti2<nti; ti2++){
+            int ti = ss_toi(words[1].c_str());
```

Comment:
`ss_toi` takes in a `const std::string&`, not a `const char*`.
```suggestion
            int ti = ss_toi(words[1]);
```